### ISSUE TITLE
.bs-alert style fixes

### ISSUE THREAD

**Issue by tomlutzenberger** [bs-detector/bs-detector#119]
# z-index
Sometimes websites use fixed or absolute headers/navs and also z-index.
Our class has a z-index of 999, whereas e.g. Bootstraps navbars use a value between 1000 - 1100.
First discovered on: www.bento.de

Suggestion:
```css
.bs-alert {
  z-index: 9999;
}
```

Realised www.focus.de:

```css
.ressort-wrapper {
  z-index: 2147483647;
}
```

Going to be hard to make this work. That number is the maximum possible integer value.

# closing button position

If pages have a vertical scrollbar, the close button gets covered.
More margin is needed. Suggestion: 
```css
.bs-alert-close {
  margin-right: 2vw;
  font-weight: bold;
}
```

**Comment by selfagency** [comment]
could probably use a `clear: both;` too

**Comment by tomlutzenberger** [comment]
Site www.focus.de:

```
.ressort-wrapper {
  z-index: 2147483647;
}
```

Going to be hard to make this work. That number is the maximum possible integer value.

**Comment by selfagency** [comment]
you could do a scan of all attribute that have that number as their value and then decrement it by 1

**Comment by tomlutzenberger** [comment]
@selfagency 
1. `clear: both` - Why?
2. Decrementing: This would be additional logic which may slow down the Ext. I'd like to prevent such things like extended website manipulation. More code == more potential bugs/problems

**Comment by selfagency** [comment]
`clear: both` because this is happening on some sites currently and is likely related to floats:

<img width="1429" alt="screenshot 2016-12-07 17 29 40" src="https://cloud.githubusercontent.com/assets/2541728/21025111/3b510154-bd55-11e6-9d05-deeacb552f38.png">

**Comment by selfagency** [comment]
in re: decrementing— what if you only targeted nav elements and elements with ids and classes containing the word nav

**Comment by tomlutzenberger** [comment]
1. That happens, because counterpunch.org targets all `p` with `max-width: 560px`. We'd better switch to `<span>`.
2. I'll take a look at this.

**Comment by selfagency** [comment]
span is an inline element and can't have text centered tho i suppose the div it's in can

**Comment by selfagency** [comment]
styles fixed in #130
